.pylintrc

```diff
@@ -0,0 +1,618 @@
+[MAIN]
+
+# Analyse import fallback blocks. This can be used to support both Python 2 and
+# 3 compatible code, which means that the block might have code that exists
+# only in one or another interpreter, leading to false positives when analysed.
+analyse-fallback-blocks=no
```

Comment:
Is this the list of default settings? This file is pretty large, so I'd prefer only non-default settings to be listed for readability

Reply:
It is default. I'm not too familiar with pylint so up to you. It was just something I added in case you wanted to add a linter. The only setting I changed was the max line length.